.github/workflows/nexus-publish-release.yml

```
@@ -0,0 +1,40 @@
+# This workflow will build a package using Maven and then publish it to
+# qbic-repo.qbic.uni-tuebingen.de packages when a release is created
+# For more information see: https://github.com/actions/setup-java#apache-maven-with-a-settings-path
+
+name: Nexus Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 1.11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.11
```

TL:DR: The suggestion keeps moving on 
```suggestion
        java-version: 1.8
```